README.md

```diff
@@ -103,7 +103,7 @@ You can interact with the InternLM Chat 7B model through a frontend interface by
 ```bash
 pip install streamlit==1.24.0
 pip install transformers==4.30.2
-streamlit run web_demo.py
+streamlit run web_demo.py -- --model_path=internlm/internlm-chat-7b-8k --max_value=8192
```

Comment:
```suggestion
streamlit run web_demo.py --model_path=internlm/internlm-chat-7b-8k --max_value=8192
```

Reply:
This is not a fault，we must add -- between web_demo.py and Args